Vagrantfile

```diff
@@ -79,6 +79,24 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  ## Provider-specific settings, libvirt
+  config.vm.provider :libvirt do |v,override|
+    override.vm.box = "fedora21"
+    override.vm.box_url = "http://citozin.com/fedora21.box"
```

Comment:
I know only 1 official image for vagrant-libvirt: beta version of fedora 22 https://getfedora.org/en/cloud/prerelease/.
I often use https://github.com/sciurus/vagrant-mutate for convert virtualbox images to libvirt.
Add links to beta - temporary solution. As soon as fedora will released it will possible set permanent link to official image.